flake.nix

```diff
@@ -0,0 +1,15 @@
+{
+  description = "Nix User Repository";
+
+  epoch = 201909;
+
+  outputs = { self, nixpkgs }:
+    {
+      overlay = final: prev: {
+        nur = import ./default.nix {
+          nurpkgs = nixpkgs;
+          pkgs = nixpkgs.pkgs;
```

Comment:
This effectively pin-points the nur to the nixpkgs version used in the repository?